packages/trpc/server/routers/viewer/attributes/bulkAssignAttributes.handler.ts

```diff
@@ -23,6 +23,9 @@ const bulkAssignAttributesHandler = async ({ input, ctx }: GetOptions) => {
     });
   }
 
+  // Create a map of attribute types for quick lookup
+  const attributeTypes = new Map<string, string>();
```

Comment:
This was much more efficient than doing it per iteration as multiple users can be assigned the same attribute